doc/man/man3/crypto_kex_x_init_client.3monocypher

```diff
@@ -0,0 +1,229 @@
+.Dd March 6, 2019
+.Dt CRYPTO_KEX_X_INIT_CLIENT 3MONOCYPHER
+.Os
+.Sh NAME
+.Nm crypto_kex_x_init_client ,
+.Nm crypto_kex_x_init_server ,
+.Nm crypto_kex_x_1 ,
+.Nm crypto_kex_x_2
+.Nd one way key exchange for establishing secure channels
+.Sh SYNOPSIS
+.In monocypher.h
+.Ft void
+.Fo crypto_kex_x_init_client
+.Fa "crypto_kex_ctx *ctx"
+.Fa "uint8_t random_seed[32]"
+.Fa "const uint8_t local_sk[32]"
+.Fa "const uint8_t local_pk[32]"
+.Fa "const uint8_t remote_pk[32]"
+.Fc
+.Ft void
+.Fo crypto_kex_x_init_server
+.Fa "crypto_kex_ctx *ctx"
+.Fa "const uint8_t local_sk[32]"
+.Fa "const uint8_t local_pk[32]"
+.Fc
+.Ft void
+.Fo crypto_kex_x_1
+.Fa "crypto_kex_ctx *ctx"
+.Fa "uint8_t session_key[32]"
+.Fa "uint8_t msg1[80]"
+.Fc
+.Ft void
+.Fo crypto_kex_x_2
+.Fa "crypto_kex_ctx *ctx"
+.Fa "uint8_t session_key[32]"
+.Fa "uint8_t remote_pk[32]"
+.Fa "conts uint8_t msg1[80]"
+.Fc
+.Sh DESCRIPTION
+These functions perform a key exchange between a
+.Em client
+and a
+.Em server
+to establish a secure channel.
+The result of the key exchange is a
+.Em session key
+that is suitable for use with
+.Xr crypto_lock 3monocypher .
+The
+.Em server
+generates an X25519 secret key and its corresponding public
+key ahead of time using
+.Xr crypto_key_exchange_public_key 3monocypher
+when the key exchange starts.
+The
+.Em client
+does not need to know the
+.Em server's
+public key before performing the key exchange.
+The
+.Em client
+initiates the key exchange.
+.Pp
+These functions differ from the
+.Xr crypto_kex_xk1_init_client 3monocypher
+functions in that the functions described in this page
+.Em do not
+authenticate the server's identity.
+If you can share a server's public key with client applications ahead of
+time,
+always prefer
+.Xr crypto_kex_xk1_init_client 3monocypher .
+.Ss Key exchange as the client
+When starting a new connection to the server, the client first reads
+the server's public key from the network;
+then it calls
```

Comment:
No, I did that to aid with diffs so that you can more clearly see what part of a sentence changed in a diff. This isn't a requirement but rather a silly habit from me writing a lot of troff.

Reply:
Okay.